### DISCUSSION TITLE
cannot login GitHub

### DISCUSSION THREAD

**Discussion by VPNAM-02** [community/community#137996] [Other Feature Feedback, Questions, & Ideas] [answered]
Help me! Now, Im lost both authentication code and recovery code. How can I restore my account with email?

**Comment by exkludera** [comment, marked as answer]
https://support.github.com/contact/cannot_sign_in

then once you're in
https://support.github.com/contact/account?contact[type]=account&flow=2fa

**Comment by queenofcorgis** [comment]
Hi @VPNAM-02, 

Thank you for your patience. The community is unable to assist with account-related situations. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
